### ISSUE TITLE
Falling through the ground and line of sight issue

### ISSUE THREAD

**Issue by blackmist03** [dalaranwow/dalaran-wow#5490]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Mobs feet were inside the ground, as well as not being able to cast on him because he was "out of range" and not in line of sight.
![wowscrnshot_052918_230151](https://user-images.githubusercontent.com/39736653/40700201-2d0c46a8-639e-11e8-85c7-e4d62c482697.jpg)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Go to the tower in sunsail Anchorage
2. Find the Wretched Hooligan on the ramp
![wowscrnshot_052918_230154](https://user-images.githubusercontent.com/39736653/40700234-56be8ec0-639e-11e8-9b37-d94d3edffb6a.jpg)
![wowscrnshot_052918_230210](https://user-images.githubusercontent.com/39736653/40700238-58fe6c3c-639e-11e8-88ac-5041d3f1a191.jpg)

3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
